templates/htan-synapse-bucket.yaml

```diff
@@ -64,6 +64,12 @@ Resources:
             AllowedOrigins: ['*']
             AllowedMethods: [GET, POST, PUT, HEAD]
             MaxAge: 3000
+      NotificationConfiguration:
+        LambdaConfigurations:
+          - Event: "s3:ObjectCreated:*"
+            Function: !Ref S3SynapseSyncFunctionRoleArn
```

Comment:
We need the ARN for the lambda function itself, rather than its role

Reply:
good catch. fixed